docs/source/_templates/docs-sidebar.html

```diff
@@ -14,3 +14,8 @@
     {% endif %}
   </div>
 </nav>
+
+
+<a class="navbar-brand" href="{{ pathto(master_doc) }}">
+  <img src="{{ pathto('_static/images/DataFusion-Logo-Background-White.png', 1) }}" class="logo" alt="logo">
```

Comment:
this used to be

```
  <img src="{{ pathto('_static/images' + logo, 1) }}" class="logo" alt="logo">
```

But then sphinx complained that `logo` wasn't defined -- so I just hard coded it to match the main logo

https://github.com/apache/arrow-datafusion/blob/af97ac886c425efefb0536c5344894703f65d7fa/docs/source/conf.py#L103